readme.txt

```diff
@@ -1,10 +1,10 @@
 === LazyLoad Plugin – Lazy Load Images, Videos, and Iframes ===
 Contributors: wp_rocket, wp_media
 Tags: lazyload, lazy load, images, iframes, thumbnail, thumbnails, smiley, smilies, avatar, gravatar, youtube
-Requires at least: 4.7
-Tested up to: 6.3
-Requires PHP: 5.6
-Stable tag: 2.3.7
+Requires at least: 4.9
+Tested up to: 6.5.5
```

Comment:
The tested up to version should match with major version, not minor.

> This field ignores minor versions, as plugins shouldn’t break with a minor update. This means a plugin only need to define the major version it is tested against and the WordPress.org plugin directory will automatically add the minor version.

Reply:
Thanks
I Made another PR for this https://github.com/wp-media/rocket-lazy-load/pull/141 @CrochetFeve0251